scripts/convert-svgs-to-icon-mixins.js

```diff
@@ -0,0 +1,59 @@
+const fs = require('fs');
+const path = require('path');
+const mime = require('mime');
+const SVGO = require('svgo');
+
+const args = process.argv;
+const symbols = /[\r\n%#()<>?\[\\\]^`{|}]/g;
+const svgo = new SVGO();
+
+function encodeSVG(data) {
+  // Use single quotes instead of double to avoid encoding.
+  data = data.replace(/"/g, "'");
+
+  data = data.replace(/>\s{1,}</g, '><');
+  data = data.replace(/\s{2,}/g, ' ');
+  data = data.replace(symbols, encodeURIComponent);
+  data = data.replace(/fill='%23(.*)' /g, "fill='#{vf-url-friendly-color($color)}' ");
```

Comment:
I think it would be worth for our future selves to include a one line comment explaining what we are replacing here and why?